src/styles/_catalog.scss

```diff
@@ -1,8 +1,9 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  grid-template-columns: repeat(4, 1fr);
```

Comment:
You can also try to set auto-fill to grid-template-columns instead of using media-queries